config/configuration.json

```diff
@@ -4,7 +4,7 @@
       "org.openstreetmap.atlas.checks.validation"
     ],
     "type": "org.openstreetmap.atlas.checks.base.BaseCheck",
-    "enabled.value.default": true
+    "enabled.value.default": false
```

Comment:
Please revert this change

Reply:
yeah sorry this happened because I had to test the output locally one last tie and forgot to revert.

Reply:
addressed